Development pb

patrickbuess opened the pull request.

patrickbuess committed:
- Variation in global trade and sectoral trade indexed at 2007, code file and figure creation file

patrickbuess committed:
- change folder name

johannesfritz committed:
reset to gta25_setup()

patrickbuess committed:
- Line 21: Change longest period from 2008-11-01 to 2009-01-01
- line 45 (and others): Expand coverage period to include all years of the period
- line 57 (and others): Get average of the coverages for alle calculated years
- repeat for every coverage function

johannesfritz commented on code/Variation in annual sectoral and global trade/code/0 Data prep.R:
```diff
@@ -0,0 +1,85 @@
+  trade.0506 <- subset(comtrade.hs12, Period < 2007)
+  trade.0506 <- aggregate(Trade.Value ~ hs+Period, trade.0506, function(x) sum(x))
+  
+  # Merge with cpc
```
be careful, duplicates could happen here

johannesfritz commented on code/Variation in annual sectoral and global trade/code/0 Data prep.R:
```diff
@@ -0,0 +1,85 @@
+  }
+  
+  # Merge trade.data with cpc
+  trade.data <- merge(trade.data, conversion, by="hs")
```
see above

johannesfritz commented on code/Variation in annual sectoral and global trade/code/0 Data prep.R:
```diff
@@ -0,0 +1,85 @@
+  
+}
+load(paste0(data.path,"Trade per sector indexed 2007.Rdata"))
+load(paste0(data.path,"Trade global indexed 2007.Rdata"))
```
not necessary

johannesfritz committed:
gta25_setup()

patrickbuess committed:
Merge branch 'master' into development-pb

patrickbuess committed:
- no changes, but github somehow wants me to commit these files again (?)

patrickbuess committed:
- no changes

patrickbuess committed:
- conflict resolving

Merge branch 'development-pb' of https://github.com/global-trade-alert/gta-25 into development-pb

# Conflicts:
#	code/Country groups targeted by destination markets/code/0 Data prep.R
#	code/Sectoral chapters panel 1/code/0 Data prep.R
#	code/USA, China and EU targeting G20/code/0 Data prep.R
#	code/What's new in the GTA/code/0 Data prep.R
#	code/What's new in the GTA/code/1 Create figures.R

patrickbuess committed:
- remove load() functions
